examples/neon-snake/src/components/BoardScreen/drawBoard.ts

```diff
@@ -5,14 +5,61 @@ import { drawDeadEnd } from "./drawDeadEnd.ts"
 import { drawAvatar } from "./drawAvatar.ts"
 import { getOptimisticStartSection } from "./getOptimisticStartSection.ts"
 
+////Draws a board showing collision grid. Useful for testing
```

Comment:
FFTI: Let's keep the function as-is and just comment out the use of it?

Reply:
Typescript throws an error:
`TS6133  is declared but its value is never read` due to `"noUnusedLocals": true,` rule being enabled in this example game. I'll just keep the function commented out.